internal/distro/test_distro/distro.go

```diff
@@ -165,6 +166,19 @@ func (t *TestImageType) Exports() []string {
 }
 
 func (t *TestImageType) Manifest(b *blueprint.Customizations, options distro.ImageOptions, repos []rpmmd.RepoConfig, packageSpecSets map[string][]rpmmd.PackageSpec, seed int64) (distro.Manifest, error) {
+	mountpoints := b.GetFilesystems()
+
+	invalidMountpoints := []string{}
```

Comment:
@ondrejbudai @teg wasn't too sure how to get around this for testing different distros. For now it's okay because the implementation for RHEL85 will come later, but would you have any tips on how I might be able to test 2 separate distros in the [`compose` tests](https://github.com/osbuild/osbuild-composer/blob/c18e2891d6ea699af4d480ef92578a7547214779/internal/client/compose_test.go#L522)? Is this even possible?

Reply:
Sorry was confusing myself here. You can ignore that comment above.